### PR TITLE
nanomsg: bump to version 1.0

### PR DESCRIPTION
**Pull request by sbinet**
This CL bumps the nanomsg version to the first production-grade
release of nanomsg: 1.0

### CODE REVIEW

File: nanomsg.sh [hunk 1]
```diff
@@ -1,13 +1,34 @@
...
+version: v1.0.0
 source: https://github.com/nanomsg/nanomsg
-tag: 0.8-beta
+tag: 1.0.0
 build_requires:
-  - autotools
+  - CMake
 ---
 #!/bin/sh
 rsync -a --delete --exclude '**/.git' $SOURCEDIR/ ./
```
Comment (ktf) [inline nanomsg.sh]: Sorry for taking long to review this. cmake should be able to build out of sources. Can you please check this is the case and use something like:

```
cmake $SOURCEDIR -DCMAKE_INSTALL_PREFIX:PATH="${INSTALLROOT}"
```

and then drop the rsync?

Comment (sbinet) [inline nanomsg.sh]: done.
PTAL

(I'll squash when LGTM'd)

**Comment by sbinet** [comment]
squashed

**Comment by ktf** [comment]
Mesos DDS issue unrelated. Merging this as `o2-daq` still compiles.

**Pull request merged**